### PR TITLE
fixing errors for Qtum APIs

### PR DESCRIPTION
**Pull request by Rachit-gaur**

### CODE REVIEW

File: qtum-connector/module/src/QtumController.ts [hunk 1]
```diff
@@ -98,24 +99,25 @@ export abstract class QtumController {
...
             return await this.service.getQtumTransactions(address, parseInt(pageSize), offset ? parseInt(offset) : undefined);
         } catch (e) {
             throw new QtumError(`Unexpected error occurred. Reason: ${e.message || e.response?.data || e}`, 'qtum.error');
         }
     }
-
-    @Get('/v3/qtum/transactions/gas/:nblocks')
+    @Get('v3/qtum/transactions/gas/:nblocks')
+    @HttpCode(HttpStatus.OK)
     async estimateFee(@Param() nblocks: number): Promise<any> {
```
Comment (samuelsramko) [inline qtum-connector/module/src/QtumController.ts:112]: This should be either @Param('nblocks') nblock: number, or @Param() params: {nblocks:number}

File: qtum-connector/module/src/QtumController.ts [hunk 2]
```diff
@@ -98,24 +99,25 @@ export abstract class QtumController {
...
             return await this.service.estimateFee(nblocks);
         } catch (e) {
             throw new QtumError(`Unexpected error occurred. Reason: ${e.message || e.response?.data || e}`, 'qtum.error');
         }
     }
-
-    @Get('/v3/qtum/transactions/gasbytes/:nblocks')
+    @Get('v3/qtum/transactions/gasbytes/:nblocks')
+    @HttpCode(HttpStatus.OK)
     async estimateFeePerByte(@Param() nblocks: number): Promise<any> {
```
Comment (samuelsramko) [inline qtum-connector/module/src/QtumController.ts:121]: This should be either @Param('nblocks') nblock: number, or @Param() params: {nblocks:number}

**Review by samuelsramko — CHANGES_REQUESTED** [review]

File: qtum-connector/module/src/QtumController.ts (hunk 2 above)
Comment (Rachit-gaur) [reply qtum-connector/module/src/QtumController.ts]: changes made

File: qtum-connector/module/src/QtumController.ts (hunk 1 above)
Comment (Rachit-gaur) [reply qtum-connector/module/src/QtumController.ts]: changes made

**Review by samuelsramko — APPROVED** [review]

**Pull request merged**